internal/api/handlers.go

```diff
@@ -109,6 +109,18 @@ func (h *Handler) ServeTableView(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (h *Handler) ServeAPISetupView(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+		return
+	}
+	w.Header().Set("Cpntent-Type", "text/html")
```

Comment:
Heh, one of the problems of copying code, is when you copy buggy code: https://github.com/Tanq16/ExpenseOwl/pull/4

Reply:
He should probably have a static handler to reduce copying.